hypershift-operator/controllers/hostedcluster/metrics/metrics.go

```diff
@@ -92,6 +92,9 @@ const (
 
 	ClusterSizeOverrideMetricName = "hypershift_cluster_size_override_instances"
 	clusterSizeOverrideMetricHelp = "Number of HostedClusters with a cluster size override annotation"
+
+	HostedClusterInfoMetricName = "hosted_cluster_info"
```

Comment:
we might want 2 info metrics
* one `hosted_cluster_info` metric for all generic labels (at least hosted cluster name, namespace, _id and maybe platform type).
* one specific for Azure platform (`hosted_cluster_azure_info`) when platform type = "Azure".

One suggestion I made somewhere else was that we could put everything into "hosted_cluster_info" but it's recommended to have consistent labels per metric. And we might get similar requests in the future for AWS, GCP, ....

Reply:
thanks @simonpasquier I'll discuss this with my team and let you know.

Reply:
@devguyio @simonpasquier at the moment we don't have any plan to revamp all metrics and I think I'd say I'm in favor of leaving this metrics as defined in the https://issues.redhat.com/browse/OCPSTRAT-2175